tasks/post_to_catalog/schemas/catalog_record_input.json

```diff
@@ -63,6 +64,10 @@
         "files"
       ],
       "properties": {
+        "providerId": {
```

Comment:
This is already present on the provider property

Reply:
but from the new cumulus schema, we have provider_id under granules table. So shouldn't this be added as well here?

Reply:
I do not believe so. I would put it in just one place. We may want to discuss changing the provider schema block to being a singular value instead of an array though. If we do, we will also need to make adjustments to copy_to_glacier as well to conform to the change.

Reply:
then I would prefer removing the provider id from here so that we don't need to modify copy_to_glacier

Reply:
removed